server/config/config.json

```diff
@@ -2,27 +2,30 @@
     "development": {
     "username": "postgres",
     "password": "12345",
-    "database": "postitDev",
+    "database": "moreRecipeDev",
     "host": "127.0.0.1",
     "port":5432,
     "dialect": "postgres"
   },
-  "test-l": {
+  "test": {
     "username": "postgres",
     "password": "12345",
-    "database": "postitTest",
+    "database": "moreRecipesTest",
     "host": "127.0.0.1",
     "port":5432,
     "dialect": "postgres",
     "logging":false
   },
-  "test": {
-    "username": "postit",
```

Comment:
Guess you copied this from a post it app?

Reply:
yeah. It's my previous project. maybe shouldn't  have copied it.